docker/.env

```diff
@@ -12,7 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DEBEZIUM_VERSION=1.1
 COMPOSE_HTTP_TIMEOUT=200
- # for big data dump set `big`, for small data dump set `small`, which is the default mode
-DATABASE_DUMP_MODE=small
\ No newline at end of file
+
+# This is for openmrs-compose.yaml; for big data dump set `big`, for small
+# data dump set `small`, which is the default mode.
+DATABASE_DUMP_MODE=small
+
+# These are for compose-controller-spark-sql.yaml
+PIPELINE_CONFIG=./config
+DWH_ROOT=./dwh
```

Comment:
nit: bring these above Database dump mode as this will likely be used more than openmrs?